modules/delegates.js

```diff
@@ -97,6 +98,32 @@ class Delegates {
 	}
 }
 
+/**
+ * Caches delegate list for last 2 rounds.
+ *
+ * @private
+ * @param {number} roundNo - Round Number
+ * @param {array} list - Delegate list
+ * @returns {Void}
+ */
+__private.updateDelegateListCache = function(roundNo, list) {
+	library.logger.debug('Updating delegate list cache.');
```

Comment:
Add round number to the `debug` message

```suggestion
	library.logger.debug('Updating delegate list cache for round', roundNo);
```